Makefile

```diff
@@ -27,6 +27,9 @@ flow:
 fix:
 	./node_modules/.bin/eslint packages/ --format=codeframe --fix
 
+prettify:
+	./node_modules/.bin/prettier --trailing-comma all --write "packages/**/src/*.js"
```

Comment:
Yeah, I can run it again with wider to see the results, some of the lines get really crowded the further you move out from 80.